control/control.yaml

```diff
@@ -16,7 +16,7 @@ components:
       properties:
         choice:
           type: string
-          enum: [config_state, port_link_state, port_capture_state, flow_transmit_state]
+          enum: [config_state, port_link_state, port_capture_state, flow_transmit_state, bgp_contol_state]
```

Comment:
bgpv4_control_state

Reply:
Agree.. Modified it accordingly.

Reply:
I have added bgpv4_control_state to control BGP related operations. These are probable step to require this options:
1. Strat all protocols and traffic. (I think existing FlowTransmitState(state='start') should handle that part)
2. Stop BGP (I have plan to trigger that through State(BgpControlState(state='stop')))
3. Flap BGP link (trigger it  using State(BgpControlState(state='link_flap')) )

I will remove this section if this is not an actual customer use case.